packages/generator/src/generator.js

```diff
@@ -16,6 +16,11 @@ export default class Generator {
 
     // Set variables
     this.isFullStatic = isFullStatic(this.options)
+    if (this.isFullStatic) {
```

Comment:
If there is already warning about legacy behavior and that should be avoided, do we still need both infos?

Reply:
this would be nice as enforcing static target because of nuxt generate with full static disabled and target static. Perhaps a hint in order to activate full static update to Nuxt v 2.14

Reply:
We don't display a warning for target server and nuxt generate since we kept the old behaviour to not-mock asyncData & fetch.